tailwind.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     colors: {
       transparent: 'transparent',
       current: 'currentColor',
+      astarblue: '#05B6FD',
```

Comment:
I thought we were not going to use tailwind and only rely on SCSS, will this show any conflicts with the style?

Reply:
@sirius651 Have you used this variable?